### PR TITLE
Modify README.md

### PR DESCRIPTION
**Pull request by jimwheaton**
Added a second line

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
 Welcome git
+Modifying readme
```
Comment (leaveofabsence) [inline README.md:2]: This should say Modify readme

File: README.md [hunk 2]
```diff
@@ -1 +1,2 @@
 Welcome git
```
Comment (leaveofabsence) [inline README.md:1]: Put hash in front of this

**Review by leaveofabsence — COMMENTED** [review]
A couple changes to do

File: README.md (hunk 1 above)
Comment (jimwheaton) [reply README.md:2]: Ok. Will make the changes

File: README.md [hunk 3]
```diff
@@ -1,2 +1,3 @@
...
 Modifying readme
+Made changes as requested
```
Comment (jimwheaton) [inline README.md]: +1 Looks good

**Pull request merged**